src/Bids/DONT.hs

```diff
@@ -0,0 +1,80 @@
+module Bids.DONT(
+    b1N  -- Re-exported from Meckwell
+  , b1NoX
+  , b1NoX2C
+  , b1No2C
+  , b1No2C2D
+  , b1No2D
+  , b1No2D2H
+  , b1No2H
+  , b1No2S
```

Comment:
Consider also having sound 3-level pre-empts, and only making situations for them when we're nonvulnerable.

Also consider making bids for when advancer breaks the "pass-or-correct" relay with their own suit.